js/index.js

```diff
@@ -0,0 +1,61 @@
+//Model
+
+var counter = 0;
+var increase = "increase";
+var decrease = "decrease";
+var reset = "reset";
+
+//View
+
+function renderCounter(number, operation) {
+	//Reset value
+	if (operation === "reset") {
+		$(".display").text("0");
```

Comment:
You could change this line to `number = 0` and then you can add `$(".display").text(number)` below the set of conditionals so you don't have to write that in each conditional statement.